easybuild/easyconfigs/r/RELION/RELION-3.0.7-foss-2019a.eb

```diff
@@ -0,0 +1,37 @@
+easyblock = 'CMakeMake'
+
+name = 'RELION'
+version = '3.0.7'
+
+homepage = 'http://www2.mrc-lmb.cam.ac.uk/relion/index.php/Main_Page'
+description = """RELION (for REgularised LIkelihood OptimisatioN, pronounce rely-on) is a stand-alone computer
+ program that employs an empirical Bayesian approach to refinement of (multiple) 3D reconstructions or 2D class
+ averages in electron cryo-microscopy (cryo-EM)."""
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+toolchainopts = {'openmp': True}
+
+source_urls = ['https://github.com/3dem/relion/archive']
+sources = ['%(version)s.tar.gz']
+checksums = ['2580d66088923a644bc7d3b02efd154b775a3ec3d010426f382bb3be5db9c98b']
+
+builddependencies = [('CMake', '3.13.3')]
+
+dependencies = [
+    ('X11', '20190311'), 
+    ('FLTK', '1.3.5'),
+    ('LibTIFF', '4.0.10'),
+    ('tbb', '2019_U4', '-GCCcore-8.2.0'),
+]
+
+configopts = "-DCMAKE_SHARED_LINKER_FLAGS='-lpthread'  -DMPI_INCLUDE_PATH=$EBROOTOPENMPI/include "
```

Comment:
For 2019.02 intel toolchain I had to use following config opts:
` -DMPI_INCLUDE_PATH=$EBROOTIMPI/intel64/include` and `"-DMPI_C_COMPILER=$EBROOTIMPI/intel64/bin/mpiicc -DMPI_CXX_COMPILER=$EBROOTIMPI/intel64/bin/mpiicpc "`